notebooks/tflite-selfie-segmentation/tflite-selfie-segmentation.ipynb

```diff
@@ -87,20 +87,23 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #17.        video_width=1280,
</pre><p>why only width? is there is plans to provide height?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2535/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>